packages/stark-ui/src/modules/collapsible/components/collapsible.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-expansion-panel class="stark-collapsible" 
+					 [ngClass]="{ 'spin' : options.spin }"
+					 id="{{ htmlId }}"
+					 [expanded]="isExpanded"
+					 hideToggle>
+	<mat-expansion-panel-header 
+			[collapsedHeight]="options.collapsedHeight"
+			[expandedHeight]="options.expandedHeight"
```

Comment:
Normally these 2 options are not needed for the `expansion-panel-header` to work... like shown in the [examples](https://material.angular.io/components/expansion/examples)